frontend/src/components/Comment/Comment.js

```diff
@@ -1,21 +1,14 @@
 import React from 'react'
-import "./Comment.scss";
-import {Link} from "react-router-dom";
-
-class Comment extends React.Component {
-    //todo: add like on comments
-    render() {
-        return (
-            <div className="comment-div">
-                <br/>
-                <Link to={`/account/${this.props.comment.account}`}>{this.props.comment.account}</Link>: {this.props.comment.comment}
-                <div className="date-created">{new Date(this.props.comment.created).toUTCString()}</div>
-                <br/>
-            </div>
-        )
-    }
-}
-
-export default Comment;
-
-
+import { Link } from "react-router-dom"
+import { useEffect } from 'react'
+import './SCSS/Comment.scss'
+export const Comment = (props) => {
+    return (
+        <div className="comment-div">
+            <br />
+            <Link to={`/account/${props.comment.account}`}>{props.comment.account} </Link> {props.comment.comment}
```

Comment:
Just making sure, the `:` after the username was intentionally left out?

Reply:
Yes, I find it more aesthetic without the `:` imo. On Instagram they don't have it too.